src/clientStub.ts

```diff
@@ -109,6 +109,18 @@ export class DgraphClientStub {
               value: "true",
             });
           }
+          if (req.readOnly) {
```

Comment:
see comment above. this would become txn options

Reply:
Seems like dgraph-js is doing it like this (storing readOnly/bestEffort in Request). Refer to here:
https://github.com/dgraph-io/dgraph-js/blob/c1ad859ea9f683ca7efb51b481e56ec868a7da1a/src/txn.ts#L79